src/main/java/com/gluonhq/substrate/model/ProjectConfiguration.java

```diff
@@ -125,10 +126,19 @@ public Path getJavaStaticLibsPath() {
      * @return the path to the JavaFX SDK (including at least the libs)
      */
     public Path getJavafxStaticPath() {
+        String depsTarget;
+        switch (targetTriplet.getOs()) {
+            case Constants.OS_LINUX: depsTarget = Constants.DEPS_TARGET_LINUX; break;
```

Comment:
why do we duplicate this? We have all info in the targetTriplet, so why do we need a "depsTarget"?